.gitignore

```diff
@@ -58,7 +58,14 @@ cpp/thirdparty/googletest/
 *.ipr
 *.iws
 
+## Datasets
 datasets/*
+!datasets/cyber.csv
+!datasets/karate-data.csv
```

Comment:
We could have a name that highlights the difference with the existing `karate.csv` otherwise there's a risk to mix them.

Reply:
yes.  I just didn't want to also edit all the test files.  there is karate renumbers, starting ID is 0, Karate-data that starts at 1, and then undirected unsymeterized.